doc/source/reliability/innodb_corrupt_table_action.rst

```diff
@@ -4,7 +4,32 @@
  Handle Corrupted Tables
 =========================
 
-Instead of crashing the server as they used to do, corrupted |InnoDB| tables are simply disabled, so that the database remains available while the corruption is being fixed.
+When a server subsystem tries to access a corrupted table,
+the server may crash.
+If this outcome is not desirable when a corrupted table is encountered,
```

Comment:
No. this is semantic linefeed. A best practice for organizing documentation sources in such a way that each clause is written on a separate line. It will not affect the output